docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.max-file-size</h5></td>
+            <td style="word-wrap: break-word;">25 mb</td>
+            <td>MemorySize</td>
+            <td>The maximum size of ForSt's file used for information logging. If the log files becomes larger than this, a new file will be created. If 0, all logs will be written to one log file. The default maximum file size is '25MB'. </td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.rescaling.use-delete-files-in-range</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>If true, during rescaling, the deleteFilesInRange API will be invoked to clean up the useless files so that local disk space can be reclaimed more promptly.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.restore-overlap-fraction-threshold</h5></td>
+            <td style="word-wrap: break-word;">0.0</td>
+            <td>Double</td>
+            <td>The threshold of overlap fraction between the handle's key-group range and target key-group range. When restore base DB, only the handle which overlap fraction greater than or equal to threshold has a chance to be an initial handle. The default value is 0.0, there is always a handle will be selected for initialization. </td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.thread.num</h5></td>
+            <td style="word-wrap: break-word;">2</td>
+            <td>Integer</td>
+            <td>The maximum number of concurrent background flush and compaction jobs (per stateful operator). The default value is '2'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.use-bloom-filter</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>If true, every newly created SST file will contain a Bloom filter. It is disabled by default.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.use-ingest-db-restore-mode</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>A recovery mode that directly clips and ingests multiple DBs during state recovery if the keys in the SST files does not exceed the declared key-group range.</td>
```

Comment:
I assume the test refers to whetehr this is true. What about when it is false. Also guidance on this would be good.

Reply:
Will go in `large_state_tuning.md`